SampleSize/SampleSizeCalculation.Rmd

```diff
@@ -0,0 +1,261 @@
+
+<!-- Figure here of clinician/statistician discussion?? Michael Coslovsky used to have a funny one... -->
+
+Speaking with a statistician about sample size calculation can be frustrating for both parties. The investigator just wants a number that they can write in their proposal/protocol/grant application. The statistician cannot give a number without various details. Hence, a good understanding from both sides is primordial.
+The aim of the study has to be well defined and all the needed assumptions educated guessed.
+
+There are two important concepts to understand before we move forward: Type 1 error and Type 2 error. 
+
+#### Type 1 error 
+
+A type 1 error is the rejection of the null hypothesis when it is actually true (i.e. a false positive - e.g. a pregnant man). This is controlled by the selection of $\alpha$, the test significance level, which is normally set at 0.05 (which represents incorrectly rejecting the null hypothesis 5% of the time). For more exploratory analyses, it might be relaxed it to 0.1. 
+
+#### Type 2 error
+
+Type 2 errors are failing to reject the null hypotheses when they is actually false (i.e. a false negative - e.g. telling a pregnant woman that is not pregnant). This is controlled by the power, which is generally set at 0.8. 
+
+
+### Two main paradigms:
+
+#### Null hypothesis significance testing (NHST) 
+
+The aim is to assess if there is a statistical difference between e.g. two groups. For example
+
+- does medication X lower Y more than medication Z
+- does surgery X results in fewer Y than surgery Z
+
+#### Precision based approaches
+The focus is rather on estimating a quantity to a given accuracy. For example, we might expect that 40% of people have a given event within a certain time frame, but we want to estimate that 40% with a specific precision, in other words within a certain interval, e.g. 5% (2.5% on either side). We are interested in the width of the confidence interval (CI). It is also possible to estimate width of the CI for differences (e.g. mean differences) and relative effects (e.g. odds ratios, relative risks).
+
+### What the statistician needs to know
+#### What is the main comparison?
+
+Between two groups is the most common (e.g. intervention vs control).
+
+#### What is the primary endpoint? 
+- what type of variable is it? Continuous? Binary? Time-to-event?
+- when is it measured? Thirty days? Six months? One year?
+
+#### What effect do you expect to see? 
+- if you have continuous data, will the difference in means be 10, 20, ..., 100?
+- if you have binary data, what proportion of events do you expect in the groups?
+  
+#### How variable will the endpoint be?
+For continuous outcomes, standard deviations are typically called for. This is less relevant if you have a binary endpoint.
+
```

Comment:
We could maybe add a point on having an idea of the ratio between the two groups to compare?

Reply:
What would you suggest precisely?